pallets/redeem/src/lib.rs

```diff
@@ -526,12 +528,12 @@ mod self_redeem {
 	) -> DispatchResult {
 		// burn the tokens that the vault no longer is backing
 		consumed_issued_tokens
-			.lock_on(account_id)
+			._lock_on(account_id)
```

Comment:
Can we find a way not to have to call functions with this underscore? I understand you created an extra trait to make this mockable. Can we move that trait next to the actual definition and unify It so that we don't need the underscore? 
Otherwise, this creates some confusion and it's not easy to see without context why this is necessary.

Reply:
I agree; in a few weeks I will have forgotten this.

I have reverted the code to allow `mocktopus` in pallet _`currency`_'s feature: _`testing-utils`_. 
That means **`cargo b --all-features`** will not work.

I created a script `build-check` to build all known packages for stable rust version. 
`testing-utils` will only compile on nightly version; or when running `cargo test`.